README.livemd

```diff
@@ -64,28 +62,6 @@ catalogue_uri = base_uri |> URI.merge("product-catalogue.json") |> URI.to_string
 catalogue = catalogue_uri |> Tesla.get!() |> Map.get(:body) |> Jason.decode!()
 ```
 
-<!-- livebook:{"output":true} -->
```

Comment:
Hey @er-jpg, thanks for the readme update.
If it's ok with you, please, keep the outputs in the readme to show how the library shapes the data.

Reply:
Thanks for reviewing!

Sorry about forgetting it, I've added in this new commit.